lib/rubysl/socket/socket.rb

```diff
@@ -261,7 +261,7 @@ class << self
 
   if RubySL::Socket.unix_socket_support?
     def self.pack_sockaddr_un(file)
-      max_path_size =  Rubinius::Config['rbx.platform.sockaddr_un.sun_path.size'] - 1
+      max_path_size =  Truffle::Support::Config['rbx.platform.sockaddr_un.sun_path.size'] - 1
```

Comment:
Can it be just `Truffle::Config` or `Truffle::NativeConfig` or `Truffle::Platform`? (I prefer `Truffle::Platform`)

Reply:
You're asking the same question for all of these...